packages/addons/service/nut/package.mk

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="nut"
+PKG_VERSION="2.8.0"
+PKG_REV="100"
+PKG_SHA256="c3e5a708da797b7c70b653d37b1206a000fcb503b85519fe4cdf6353f792bfe5"
+PKG_ARCH="any"
+PKG_LICENSE="GPLv2+"
+PKG_SITE="https://github.com/networkupstools/nut"
+PKG_URL="https://github.com/networkupstools/${PKG_NAME}/releases/download/v${PKG_VERSION}/nut-${PKG_VERSION}.tar.gz"
```

Comment:
No need for `${PKG_NAME}` if you're hardcoding `nut` in the .tar.gz section. Use `nut` again here.

Reply:
It's just a matter of habit since maintaning `PKGBUILD`s